clap_builder/src/builder/arg.rs

```diff
@@ -3908,6 +3908,21 @@ impl Arg {
         Some(shorts)
     }
 
+    /// Get hidden short aliases for this argument, if any
+    #[inline]
+    pub fn get_hidden_short_aliases(&self) -> Option<Vec<char>> {
```

Comment:
These function names should match the property its reading (`Arg::short_aliases`)

Reply:
In this PR, `get_hidden_short_aliases` should no longer be needed. To support [this comment on issue 3951](https://github.com/clap-rs/clap/issues/3951#issuecomment-2210951383), we only need this code snippet (https://github.com/clap-rs/clap/blob/ff204809a5ac1e9d2522b3375708624ad2ce6c6f/clap_builder/src/builder/arg.rs#L3890). 
So i will remove this function.

Reply:
`get_hidden_aliases` is still present and this applies to it as well

Reply:
`get_hidden_aliases` is a getter for aliases of argument which is not visible. It's used for supporting to complete `hidden aliases` of long options. Now that we have `get_visible_aliases` and `get_all_aliases`, we can indirectly get `hidden_aliases` through these two methods. However, adding `get_hidden_aliases` is a simpler way.

Reply:
I wasn't commenting on whether it  should be there but on the original issue raised here

> These function names should match the property its reading 

There is no `Arg::hidden_aliasses`, instead its `Arg::aliases`.  The getter should then be `Arg::get_aliases`